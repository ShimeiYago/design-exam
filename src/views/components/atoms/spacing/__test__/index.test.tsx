import { shallow, ShallowWrapper } from 'enzyme';
import { Spacing, Props } from '..';

let wrapper: ShallowWrapper<Props>;

const props: Props = {
  children: 'text',
  margin: '10px',
};

describe('Shallow Snapshot Tests', () => {
  beforeEach(() => {
    wrapper = shallow(<Spacing {...props} />);
  });

  it('basic', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
