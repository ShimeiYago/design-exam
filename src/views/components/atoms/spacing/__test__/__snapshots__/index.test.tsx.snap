// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shallow Snapshot Tests basic 1`] = `
<div
  style={
    Object {
      "marginBottom": "10px",
      "marginLeft": undefined,
      "marginRight": undefined,
      "marginTop": "10px",
      "paddingBottom": undefined,
      "paddingLeft": undefined,
      "paddingRight": undefined,
      "paddingTop": undefined,
    }
  }
>
  text
</div>
`;
